src/content/docs/en/guides/integrations-guide/lit.mdx

```diff
@@ -126,6 +126,34 @@ import { MyElement } from '../components/my-element.js';
 Lit requires browser globals such as `HTMLElement` and `customElements` to be present. For this reason the Lit renderer shims the server with these globals so Lit can run. You *might* run into libraries that work incorrectly because of this.
 :::
 
+### Use TypeScript
```

Comment:
I'm just checking here whether this section is really about TypeScript support, or about using experimental decorators, which happen to require this TS configuration.

It's possible that this section is instead:

```suggestion
### Experimental Decorators
```

?

Reply:
Yeah. It's only about decorators, you can use `.ts` file without decorators.